distribution/src/conf/internal-apis.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!--
+    This is the configuration file for internal APIs
+-->
+
+<apis>
+    <api name="MetricAPI" class="org.wso2.carbon.prometheus.publisher.service.MetricAPI"/>
```

Comment:
Should it be Prometheus API?

Reply:
Yes , I'll rename this . anyway internally in mediation this is implemented as MetricAPI